### ISSUE TITLE
[BUG] OpenSource Version is now 2.0

### ISSUE THREAD

**Issue by KnowledgeGarden** [opensearch-project/documentation-website#648]
### What is the bug?
It's not so much a bug as potential bugs if you move from OS version < 2.0 to the new 2.0 because they have a new REST client

### How can one reproduce the bug?
Can't without changing `opensource.version' value in module.xml

### What is the expected behavior?
I can only predict that REST client code might have to change

**Comment by Naarcha-AWS** [comment]
@KnowledgeGarden: Could you provide more specific on what you mean? Trying to decide how we would adjust the documentation to account for REST changes. 

Furthermore, what differences in behavior did you encounter before upgrading?

**Comment by KnowledgeGarden** [comment]
I did not upgrade. I am building a de novo platform and noticed that the 1+ REST platform directs to the 2+ platform. I had already followed the instructions for adding shards and replicas values to a Settings builder; when I switched to the new maven dependencies, that builder was no longer available; I could not import that Settings class. I cannot find a similar class in the 2+ platform.

It seems to me that the upgrade from 1+ to 2 is a build breaker since important features got refactored out entirely, not just moved. 2 seems to be a new design; perhaps improved, I just don't know.

I did find an example where I can use the mappings file to add those two settings; I can live with that.
What I find problematic is that the documentation does not cover those two settings very well.

The term "latest" is used in online documentation. Unfortunately, google still directs to the 1+ documentation as latest; it takes more digging to find the 2+ documentation.

**Comment by KnowledgeGarden** [comment]
ExistsRequest is found in the opensearch-java dependency, which I loaded.
Now, maven talks about
opensearch-rest-high-level-client
and
opensearch-rest-client
both at version 2.0, and each with a different maven dependency
The instructions I followed called for opensearch-rest-client, which is the cause of loss of shards and replicas APIs which were there when I installeid opensearch-rest-high-level-client. I abandoned that when I stumbled on an opensearch document (don't recall which) which said that the high-level was depricated in favor of the rest-client.
According to the instruction page, I installed opensearch-rest-client and opensearch-java.
I am still no closer to using ExistsRequest.
